lib/mix/tasks/gen/ash.gen.resource.ex

```diff
@@ -416,6 +416,51 @@ if Code.ensure_loaded?(Igniter) do
     defp add_relationships_to_resource(igniter, resource, options) do
       Enum.reduce(options[:relationship] || [], igniter, fn relationship, igniter ->
         case String.split(relationship, ":") do
+          ["many_to_many", name, destination, "through", through_destination | modifiers] ->
```

Comment:
Hmm...so this is close, but there is a bit of an issue with it. We don't require that modifiers be ordered in any particular way. This shouldn't be considered a modier, instead it should just be a requirement that you do `many_to_many:comments:MyApp.Blog.PostComment:MyApp.Blog.Comment:...<other_modifiers>`. Then its not a modifer, just an extra positional element.

Reply:
Makes sanse. So I should just update the docs or...?

Reply:
oh I see, raising an error when it's not provided

Reply:
I don't think we support it yet, so you'd have to add the matcher for many to many relationships and that `through` option. Just saying thats how we should do it instead of `through:Through`